RDKOSS-128: Remove unmapped binaries from OSS packages

h1712 opened the pull request:
Reason for change: We have lot of unmapped binaries coming from OSS packages. In this PR, we have selectively removed the binaries that are unused in RDK and also performed a regression test. With no failures, proceeding further for the merge.
Test Procedure: Ensure all the RDK features are working as expected.
Note: In future, if the package is required, feel free to remove it from "extras" package which will bring it in the default package.

h1712 committed:
RDKOSS_128-test: remove unmapped binaries

h1712 committed:
Update libcroco_%.bbappend

h1712 committed:
Update zstd_%.bbappend

h1712 committed:
Update libpcre_8.39.bb

h1712 committed:
Update libpcre_8.39.bb

h1712 committed:
Delete recipes-extended/zstd directory

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 2 **violation**s, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/meta-rdk-oss-reference/137/rdkcentral/meta-rdk-oss-reference

* Commit: 1642cd48b1bf105d7845df53d275f4685528d08f

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/61a56337f5543887d7d84148718ccf31)'

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: 1642cd48b1bf105d7845df53d275f4685528d08f
'

h1712 committed:
RDKTV-37484: Add libraries required for iptables service (#145)

Reason for the change: Add missing iptable packages required for rdk platform

Arjun-DR committed:
RDKEVD-2071: wipefs and blkdiscard are required (#142)

Reason for change: Bring in wipefs and blkdiscard as they are required for few platforms while performing factory reset.
Also update "util-linux-" to ${PN} to support the MLPREFIX as well.

Signed-off-by: Arjun <arjun_daasuramdass@comcast.com>
Co-authored-by: maniselva006c <mani.sselvaraj@gmail.com>

maniselva006c committed:
RDKTV-37484: Add libraries required for iptables service (#150)

Reason for change: Update the util-linux package revision

maniselva006c committed:
RDKEVD-2071: wipefs and blkdiscard are required (#149)

Reason for change: Update the util-linux package revision

Commit:
RDKE-872: Updated Packagegroup revision to 4.7.1

Reason for change: Update packagegroup revision to reflect new version

Signed-off-by: mselva006c <mani_selvaraj@comcast.com>

Commit:
RDKE-872: Update change log for 4.7.1 release

Reason for change: Update change log for 4.7.1 release

Signed-off-by: mselva006c <mani_selvaraj@comcast.com>

Commit:
Merge branch 'hotfix/4.7.1' into support/4.7.0

h1712 committed:
Merge pull request #174 from rdkcentral/feature/RDKOSS-128-rebase

REBASE with 4.7.1

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/meta-rdk-oss-reference?pullRequest=137) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/meta-rdk-oss-reference?pullRequest=137) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: h1712<br/>:white_check_mark: Arjun-DR<br/>:white_check_mark: maniselva006c<br/>:x: mselva006c<br/><hr/>**mselva006c** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/meta-rdk-oss-reference?pullRequest=137) it.</sub>
